.github/pull_request_template.md

```diff
@@ -9,8 +9,12 @@ See here for more information and a list of available options:
 http://tedana.readthedocs.io/en/latest/contributing.html#pull-requests
 -->
 
+<!-- Please indicate after the # which issue you're closing with this PR -->
```

Comment:
:heart_eyes: This is great ! Should we explain that this will automatically close the issue, maybe by including [this link](https://help.github.com/articles/closing-issues-using-keywords/) ?